conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -40,9 +41,8 @@ const (
 	followerMode
 )
 
-// Retry
-const (
-	retries = 5
+var (
+	waitForRoundTimeout = 5 * time.Second
```

Comment:
Nit: 

A more conservative timeout would be 45 seconds. I agree that we want to endow conduit with a greater amount of determinism regarding the outcome of each call to the `waitForBlock` endpoint. So it's a good idea to make the call to the endpoint timeout on _it's own terms_ rather than the endpoint's as is being done in the PR. On the other hand, we might still want the ability to keep 10 threads of the algod importer running concurrently after we've all caught up, and a 45 sec timeout would allow for that. If we narrow the timeout to 5 secs, we essentially only allow one or two algod importer threads to run at a time (probably only one due to round time variability).

On the other hand, we can change the value as aggressively as in the PR, and if the need arises in the future to raise it back to 45 secs we can do it.

Reply:
A worthwhile thought for a future PR or even the pipelining effort.

Reply:
https://github.com/algorand/conduit/pull/125